pkg/storaged/pages.jsx

```diff
@@ -528,62 +528,19 @@ export const PageTable = ({ emptyCaption, aria_label, pages, crossrefs, sorted,
                 size = <StorageSize size={size} />;
         }
 
-        function onClick(event) {
-            if (!event || event.button !== 0)
-                return;
-
-            if (page.location)
-                cockpit.location.go(page.location);
-        }
-
-        function is_clickable(element) {
-            return element.classList.contains("pf-m-clickable");
-        }
-
-        function next_clickable_sibling(element) {
-            do {
-                const next = element.nextElementSibling;
-                if (next && is_clickable(next))
-                    return next;
-                element = next;
-            } while (element);
-
-            return null;
-        }
-
-        function previous_clickable_sibling(element) {
-            do {
-                const prev = element.previousElementSibling;
-                if (prev && is_clickable(prev))
-                    return prev;
-                element = prev;
-            } while (element);
-
-            return null;
-        }
-
-        function onRowKeyDown(event) {
-            const { code, target } = event;
-
-            if (target.nodeName == "TR") {
-                if (code == "Space" || code == "Enter") {
-                    if (page.location)
-                        cockpit.location.go(page.location);
-                    event.preventDefault();
-                }
-                if (code == "ArrowDown") {
-                    const next = next_clickable_sibling(target);
-                    if (next)
-                        next.focus();
-                    event.preventDefault();
-                }
-                if (code == "ArrowUp") {
-                    const prev = previous_clickable_sibling(target);
-                    if (prev)
-                        prev.focus();
-                    event.preventDefault();
-                }
-            }
+        function location_link(location) {
+            if (!location)
+                return null;
+            if (typeof location == "string")
+                return location;
+            if (!location.to)
+                return location.label;
+
+            return (
+                <Button isInline variant="link" onClick={() => cockpit.location.go(location.to)}>
```

Comment:
Part of test suggestion in another thread. For suggestion see [Button component props](https://www.patternfly.org/components/button/#button)
```suggestion
                <Button isInline variant="link" onClick={() => cockpit.location.go(location.to)} ouiaId={location.to}>
```
(Haven't tested this suggestion)